bam-cmd/src/main/java/org/victorchang/QnameCommand.java

```diff
@@ -54,18 +54,20 @@ static Path getDefaultIndexPath(Path bamPath) {
     @Parameters(paramLabel = "bam-file", description = "Path to the BAM file")
     Path bamPath;
 
-    @Option(names = {"-i", "--index-path"}, description = "Directory to store index files. By default uses a directory name that starts with the BAM file name (so stored next to it).")
+    @Option(names = {"-i", "--index-path"}, description = "Directory to store index files. By default uses a directory name that starts with the BAM file name (so stored next to it)")
     Path indexDirectory;
     @Option(names = "--thread-count", description = "Number of threads used for sorting", defaultValue = "1")
     int threadCount;
     @Option(names = "--sort-buffer-size", description = "Maximum number of records per buffer used for sorting", defaultValue = "500000")
     int sortBufferSize;
     @Option(names = {"-l", "--limit-bytes"}, description = "Only read and index first given bytes")
     long bytesLimit;
-    @Option(names = {"-t", "--temporary-path"}, description = "Directory to store temporary files for sorting. By default uses the index directory.")
+    @Option(names = {"-t", "--temporary-path"}, description = "Directory to store temporary files for sorting. By default uses the index path")
     Path tempDirectory;
     @Option(names = {"-v", "--verbose" }, description = "Switch on verbose output", defaultValue = "false")
     boolean verbose;
+    @Option(names = {"--force" }, description = "Overwrite existing index", defaultValue = "false")
```

Comment:
Minor: this could be any existing directory, so we may not be overwriting the index.

Reply:
in this case , it just overwrite these qname.0, qname.1 files in that directory